engine/src/main/java/org/hibernate/search/bridge/builtin/TikaBridge.java

```diff
@@ -210,13 +210,16 @@ private FileInputStream openInputStream(File file) {
 	}
 
 	private static class AutoDetectParserProvider implements TikaParserProvider {
+		private static AutoDetectParserProvider INSTANCE = new AutoDetectParserProvider();
+
```

Comment:
You invoking Sanne made me think maybe it's not such a good idea considering you are throwing it just after you used it and it's not kept at runtime.

It might make more sense for the other 2.